spec/unit/unparser_spec.rb

```diff
@@ -191,6 +191,7 @@ def assert_generates_from_ast(parser, ast_with_comments, expected)
       generated = Unparser.unparse(*ast_with_comments).chomp
       expect(generated).to eql(expected)
       ast, comments = parse_with_comments(generated)
+      expect(ast).to eq(ast_with_comments.first)
```

Comment:
We should not use `eq` matcher as this only compares via `#==` that may coerce, `#eql?` is the better non coercing operator, so `eql` rspec syntax.

Reply:
Thanks. Addressed.